### PR TITLE
Add Smartlands validator

### PR DESCRIPTION
**Pull request by MikhailTamarin**

### CODE REVIEW

File: common/nodes.js [hunk 1]
```diff
@@ -265,6 +265,13 @@ module.exports = [
     port: 11625,
     publicKey: "GCWJKM4EGTGJUVSWUJDPCQEOEP5LHSOFKSA4HALBTOO4T4H3HCHOM6UX"
   },
+  {
+    id: "smartlands",
+    name: "Smartlands Platform",
+    host: "stellar.smartlands.io",
+    port: 11625,
+    publicKey: "GCBAQCWXKIIOQOEDIW6EB7GY6UREHPTKK26NZFLVK63ZTRFSXDVX2AHV"
```
Comment (bartekn) [inline common/nodes.js]: Make sure the public key is visible in smartlands.io [stellar.toml](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0001.md) file in `OUR_VALIDATORS` section.

File: common/nodes.js [hunk 2]
```diff
@@ -265,6 +265,13 @@ module.exports = [
...
+  },
```
Comment (bartekn) [inline common/nodes.js]: Add this as the last entry in this file, with the `,` appended.

File: common/nodes.js (hunk 1 above)
Comment (MikhailTamarin) [reply common/nodes.js]: @bartekn added public key to OUR_VALIDATORS section at https://smartlands.io/.well-known/stellar.toml

File: common/nodes.js (hunk 2 above)
Comment (MikhailTamarin) [reply common/nodes.js]: @bartekn added this as the last entry in this file, with the , appended.

**Pull request merged**